website/content/docs/configuration/controller.mdx

```diff
@@ -29,6 +30,9 @@ controller {
   - `url` - Configures the URL for connecting to Postgres
   - `migration_url` - Can be used to specify a different URL for migrations, as that
     usually requires higher privileges.
+  - `max_open_connections` - Can be used to control the maximum number of connections that can be opened by the controller.
```

Comment:
Perhaps update this to read:

 - `max_open_connections` - Can be used to control the maximum number of
  connections that can be opened by the controller. The minimum number of
  connections required is 2. Every controller requires 1 non-pooled
  shared lock connection (which is held for the lifetime of the controller) and
  a minimum of 1 pooled connection which is used for all other database
  requests. Setting this value to 0 will allow the controller to open as many
  connections as  needed.

Reply:
Thanks for the feedback @jimlambrt have updated the docs with the specified description